.yamato/ruamel/editor_pinning/merge_revisions.py

```diff
@@ -46,24 +46,21 @@ def apply_changes(path):
         return False
     
     changed_editor = apply_changes(editor_versions_file)
-    changed_yml = apply_changes(yml_files_path)
+    #changed_yml = apply_changes(yml_files_path)
```

Comment:
Should this be commented?

Reply:
Yes, so that the merge revisions job only merges the editor versions file and not the .ymls themselves (which are merged by the merge all job), but left it as a comment in case we need to revert back to it at some point